charts/duplicati/README.md

```diff
@@ -0,0 +1,109 @@
+# Duplicati Backup Client
+
+This is a helm chart for [duplicati](https://github.com/duplicati/duplicati) leveraging the [Linuxserver.io image](https://hub.docker.com/r/linuxserver/duplicati/)
+
+## TL;DR;
+
+```shell
+$ helm repo add billimek https://billimek.com/billimek-charts/
+$ helm install billimek/duplicati
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/duplicati
```

Comment:
With helm3 being the default now, we can probably drop the `--name` flag for the `helm install` command.